GameOfLife/Calculations/RandomCellGenerator.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+
+    public class RandomCellGenerator
+    {
+        // Generates randomly which cells are alive which not
+        public int[,] RandCells(int[,] GameField)
```

Comment:
int[,] GameField - method parameters should start with lower case letter, so int[,] gameField.

Check for other methods as well

Reply:
Checked all methods and fixed.